charts/osm/crds/meshconfig.yaml

```
@@ -84,6 +84,10 @@ spec:
                           description: "Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/"
                           type: object
                           additionalProperties: true
+                    configResyncInterval:
+                      description: Resync interval for regular proxy broadcast updates
+                      type: string
+                      default: "0"
```

For consistency with default in the chart
```suggestion
                      default: "0s"
```